core/modules/system/config/system.core.json

```
@@ -92,6 +92,7 @@
     "user_mail_status_canceled_notify": 0,
     "user_register": "admin_only",
     "user_signatures": 0,
+    "user_login_appearance_setting": "simplified",
```

There's no need to put "setting" in the name of a config value. Could you find/replace this entire PR with `user_login_appearance_setting` getting shortened to `user_login_appearance`?

```suggestion
    "user_login_appearance": "simplified",
```

---

I made this change across the PR.